addons/mail/models/mail_message.py

```diff
@@ -979,19 +979,9 @@ def _message_format(self, fnames):
             main_attachment = self.env['ir.attachment']
             if message_sudo.attachment_ids and message_sudo.res_id and issubclass(self.pool[message_sudo.model], self.pool['mail.thread']):
                 main_attachment = self.env[message_sudo.model].sudo().browse(message_sudo.res_id).message_main_attachment_id
-            attachment_ids = []
-            for attachment in message_sudo.attachment_ids:
-                attachment_ids.append({
-                    'checksum': attachment.checksum,
-                    'id': attachment.id,
-                    'filename': attachment.name,
-                    'name': attachment.name,
-                    'mimetype': 'application/octet-stream' if safari and attachment.mimetype and 'video' in attachment.mimetype else attachment.mimetype,
-                    'is_main': main_attachment == attachment,
-                    'res_id': attachment.res_id,
-                    'res_model': attachment.res_model,
-                })
-
+            attachment_ids = message_sudo.attachment_ids._attachment_format()
```

Comment:
This was a bad variable name, as it doesn't contain id of attachments (not your fault, but I guess it might be appropriate to change it now).
Suggestion: simply `attachments`, or something similar but with the `format` keyword on it like `attachment_formats`?